src/util.js

```diff
@@ -0,0 +1,5 @@
+import { ENDPOINTS } from './config';
+
+export const isReferredFromAccountPages = (referrer, env) => {
+    return referrer && (referrer.startsWith(ENDPOINTS.BFF[env]) || referrer.startsWith(ENDPOINTS.SPiD[env]));
```

Comment:
I'm not sure if this is a general rule that will always be true, but `ENDPOINTS.SPiD` would match on anything `ENDPOINTS.BFF` matches on, so it might be unnecessary to check both?
https://github.com/schibsted/account-sdk-browser/blob/master/src/config.js#L46-L59
_(`ENDPOINTS.BFF` seems identical to `ENDPOINTS.SPiD`, except it also ends with `/authn/`)_

Reply:
We assumed they might diverge in the future 🤷 
We also considered adding the `SESSION_SERVICE` there too, because it wouldn´t hurt, like:

`return referrer && Object.keys(ENDPOINTS).some(endpoint => referrer.startsWith(ENDPOINTS[endpoint][env])`